internal/server/server.go

```diff
@@ -25,7 +25,7 @@ func Server() {
 	r.HandleFunc("/searchusers", Serviceuser.SearchUsers).Methods("GET")
 	r.HandleFunc("/searchtweet", Tweets.SearchTweets).Methods("GET")
 	r.HandleFunc("/getfollowingtweet", Tweets.GetFollowingTweets).Methods("GET")
-	r.HandleFunc("/tweets_create", Tweets.CreateTweet).Methods("POST")
+	http.Handle("/createtweet", Serviceuser.AuthHandler(http.HandlerFunc(Tweets.CreateTweet)))
```

Comment:
https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5